src/PAL/COM/sockets/ssl/MbedTLS/ssl_generic_init_internal.cpp

```
@@ -166,13 +171,21 @@ bool ssl_generic_init_internal(
         }
         else if (sslMode & SslProtocols_Tls13)
```

Clang (14.0.6)

[note] +1, nesting level increased to 2

(clang)